test/e2e/clusterctl_upgrade_test.go

```diff
@@ -26,11 +26,96 @@ import (
 var _ = Describe("When testing clusterctl upgrades [clusterctl-Upgrade]", func() {
 	ClusterctlUpgradeSpec(ctx, func() ClusterctlUpgradeSpecInput {
 		return ClusterctlUpgradeSpecInput{
-			E2EConfig:             e2eConfig,
```

Comment:
Looks like there's a diff between the way this is in 1.2 vs 1.3 - I think the solution is to stick with whatever version is current in v1.2 - it looks like there's no Kubernetes version changes needed here?

Reply:
Yep!